benches/bench.rs

```diff
@@ -47,6 +47,13 @@ pub fn http_requests(crit: &mut Criterion) {
 	run_concurrent_round_trip(&rt, crit, client.clone(), "http_concurrent_round_trip");
 }
 
+pub fn batched_http_requests(crit: &mut Criterion) {
+	let rt = TokioRuntime::new().unwrap();
+	let url = rt.block_on(helpers::http_server());
+	let client = Arc::new(HttpClientBuilder::default().build(&url).unwrap());
+	run_round_trip_with_batch(&rt, crit, client.clone(), "batched_http_round_trip", 10);
```

Comment:
would be nice to with different batch sizes just to see how it performs with bigger input ^^

Reply:
Done in 78f3b25c3a02277153f2c85e9f6e1e712296faeb